fuse/data/tokenizers/modular_tokenizer/inject_utils.py

```diff
@@ -47,7 +45,7 @@ class InjectorToModularTokenizerLib:
     @staticmethod
     def build_placeholder_meta_tokenization(
         *,
-        sequence: Union[str, list, tuple],
+        sequence: str,
```

Comment:
Why did you change it if you still support those types?

Reply:
I just copied over the code, so probably you already fixed this and it got overriden.
fixed it now to contain the union.